### PR TITLE
Change the name of test project for better readability

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
The name of test project gets a FQN of the class. This looks strange in the test. 
**See screenshot:**
ugly_name
The PR fix it. The name will be build from simple name of the class + 2 random symbols

### CODE REVIEW

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java [hunk 1]
```diff
@@ -44,10 +43,10 @@
 /** @author Musienko Maxim */
 public class RenameTypeTest {
   private static final Logger LOG = LoggerFactory.getLogger(RenameTypeTest.class);
-  private static final String nameOfProject =
-      NameGenerator.generate(RenameTypeTest.class.getName(), 2);
+  private static final String NAME_OF_PROJECT =
+      NameGenerator.generate(RenameTypeTest.class.getSimpleName(), 2);
   private static final String pathToPackageInChePrefix =
-      nameOfProject + "/src" + "/main" + "/java" + "/renametype";
+      NAME_OF_PROJECT + "/src" + "/main" + "/java" + "/renametype";
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java]: `NAME_OF_PROJECT + "/src" + "/main" + "/java" + "/renametype"`
>
`NAME_OF_PROJECT + "/src/main/java/renametype"`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java [hunk 2]
```diff
@@ -44,10 +43,10 @@
 /** @author Musienko Maxim */
 public class RenameTypeTest {
   private static final Logger LOG = LoggerFactory.getLogger(RenameTypeTest.class);
-  private static final String nameOfProject =
-      NameGenerator.generate(RenameTypeTest.class.getName(), 2);
+  private static final String NAME_OF_PROJECT =
+      NameGenerator.generate(RenameTypeTest.class.getSimpleName(), 2);
   private static final String pathToPackageInChePrefix =
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java]: This is a constant as well. Name should be in upper case.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java [hunk 3]
```diff
@@ -70,17 +69,17 @@ public void setup() throws Exception {
...
     projectExplorer.quickExpandWithJavaScript();
     loader.waitOnClosed();
   }
 
   @BeforeMethod
-  public void setCurrentFieldForTest(Method method) throws IOException {
+  public void setCurrentFieldForTest(Method method) {
     try {
       setFieldsForTest(method.getName());
     } catch (Exception e) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java]: It's strange - we are hiding exception here. Is it really correct behavior?

Comment (musienko-maxim) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java]: As i guess this have been done for avoiding with skipping in the `@Before` method annotation. But i can not remember the exact cause

**Comment by dmytro-ndp** [comment]
ci-build

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**